src/components/enter-mfa/enter-mfa-controller.ts

```diff
@@ -27,6 +27,19 @@ export function enterMfaGet(
   return async function (req: Request, res: Response) {
     const isAccountRecoveryEnabledForEnvironment = supportAccountRecovery();
 
+    if (
```

Comment:
This won't only show if support2hrLockout() is true so counts as a change to live. Is that correct?

Reply:
My intuition is that this code should be here, but I'll wrap it in a feature switch just to be sure